src/pivotal-ui/components/grids/grids.scss

```diff
@@ -2264,3 +2264,525 @@ page](/containers.html).
 .container-fullwidth {
  max-width: initial !important;
 }
+
+/*doc
+---
+title: Flex Grids
+name: flexgrid
+parent: grid
+---
+
+```html_example
+<div class="frow mbxl">
+ <div class="fitem-xs">
+  <div class="bg-neutral-10">
+   &nbsp;
+  </div>
+ </div>
+</div>
+<div class="frow mbxl">
+ <div class="fitem-xs">
+  <div class="bg-neutral-10">
+   &nbsp;
+  </div>
+ </div>
+ <div class="fitem-xs">
+  <div class="bg-neutral-10">
+   &nbsp;
+  </div>
+ </div>
+</div>
+<div class="frow mbxl">
+ <div class="fitem-xs">
+  <div class="bg-neutral-10">
+   &nbsp;
+  </div>
+ </div>
+ <div class="fitem-xs">
+  <div class="bg-neutral-10">
+   &nbsp;
+  </div>
+ </div>
+ <div class="fitem-xs">
+  <div class="bg-neutral-10">
+   &nbsp;
+  </div>
+ </div>
+</div>
+```
+
+Holy grail layout:
+
+```html_example
+<div class="frow flex-justify-center-xs">
```

Comment:
I've expanded the words in this example ... will that suffice? I'm not sure `html_example_table` is clearer in this case, due to the boilerplate.